plugin.js

```diff
@@ -114,8 +115,13 @@ class CspHtmlWebpackPlugin {
             .map((i, element) => this.hash($(element).html()))
             .get();
 
-          policyObj['script-src'] = policyObj['script-src'].concat(inlineSrc);
-          policyObj['style-src'] = policyObj['style-src'].concat(inlineStyle);
+          // Wrapped in flatten([]) to handle both when policy is a string and an array
```

Comment:
Two alternative ways to solve the issue without relying on `lodash/flatten` are:

```javascript
policyObj['script-src'] = Array.isArray(policyObj['script-src'])
  ? policyObj['script-src'].concat(inlineSrc)
  : [policyObj['script-src'], inlineSrc];
policyObj['style-src'] = Array.isArray(policyObj['style-src'])
  ? policyObj['style-src'].concat(inlineStyle)
  : [policyObj['style-src'], inlineStyle];
```

or

```javascript
// Somewhere in plugin.js, e.g. line 19.
// Couldn't come up with a good name for this function :joy:
function ensureIsArray(strOrArray) {
  return Array.isArray(strOrArray) ? strOrArray : [strOrArray];
}

// and at line 119
policyObj['script-src'] = ensureIsArray(policyObj['script-src']).concat(inlineSrc);
policyObj['style-src'] = ensureIsArray(policyObj['style-src']).concat(inlineStyle);
```